PigLatin/PigLatin.cs

```diff
@@ -6,16 +6,48 @@ class Program
     {
         public static void Main()
         {
-            // your code goes here
+            /*  any consonants before first vowel get moved to end
+                if updated word ends in vowel add "yay"
+                if word ends in consonant add "ay"
+                fox -> oxfay
+                tsk -> tskay
+                are -> areyay
+                mine -> inemay
+                thing -> ingthay
+                extra credit treat y as vowel when in middle of word
+                treat y as consonant
+            */
 
-            // leave this command at the end so your program does not close automatically
+            Console.WriteLine("I convert words to pig latin, what word would you like me to translate?");
+            string userInput = Console.ReadLine().ToLower();
+
+            char [] vowels = {'a', 'e','i','o','u'};
+
+            int firstVowelIndex = userInput.IndexOfAny(vowels);
+            int firstYIndex = userInput.IndexOf('y');
+
+            string firstPart = userInput.Substring(0,firstVowelIndex);
```

Comment:
this will throw an exception if the word does not have a vowel or  does not have a Y.
Also the logic does not correctly handle letters that start and end with a vowel.